src/main/java/net/dries007/tfc/objects/blocks/BlocksTFC.java

```diff
@@ -396,7 +396,8 @@ public static void registerBlocks(RegistryEvent.Register<Block> event)
         // todo: sluice
         // todo: quern
         // todo: loom
-        // todo: bellows
+        inventoryItemBlocks.put(register(r, "bellows", new BlockBellows(), CT_MISC), ItemBlockTFC.class);
+        ;
```

Comment:
I really don't get why IJ did place that semicolon...

Reply:
Don't review your own PR... there is no point. Just fix what needs fixing.

Reply:
Do you really want me to do an entire commit just to erase that semicolon?